content/rs/installing-upgrading/_index.md

```diff
@@ -70,42 +57,40 @@ Before installing, review these notes:
 
 - Review the [security considerations]({{< relref "/rs/security/" >}}) for your deployment.
 
-- If you want to use Redis on Flash (RoF) for your databases, review the [prerequisites, storage requirements, and other considerations]({{< relref "/rs/concepts/memory-architecture/redis-flash.md" >}}) for RoF databases and prepare and format the flash memory.
-    {{% expand "To prepare and format the flash memory:" %}}
-Run:
+- If you want to use Redis on Flash (RoF) for your databases, review the prerequisites, storage requirements, and [other considerations]({{< relref "/rs/concepts/memory-architecture/redis-flash.md" >}}) for RoF databases and prepare and format the flash memory.
 
-```sh
-sudo /opt/redislabs/sbin/prepare_flash.sh
-```
+    Use the `prepare_flash` script to prepare and format flash memory:
 
-This command finds all of the unformatted disks and mounts them as RAID partitions in `/var/opt/redislabs/flash`.
+    ```sh
+    sudo /opt/redislabs/sbin/prepare_flash.sh
+    ```
 
-To verify the disk configuration, run:
+    This script finds unformatted disks and mounts them as RAID partitions in `/var/opt/redislabs/flash`.
 
-```sh
-sudo lsblk
-```
-    {{% /expand %}}
+    To verify the disk configuration, run:
+
+    ```sh
+    sudo lsblk
+    ```
 
 - [Disable Linux swap]({{< relref "/rs/installing-upgrading/configuring/linux-swap.md" >}}) on all cluster nodes.
-- Make sure that you have root-level access to each node, either directly or with sudo.
+- Make sure that you have root-level access to each node, either directly or with `sudo`
```

Comment:
Maybe explain why? Ease customers' mind a little bit by saying there are some operations during the installation that require it. Big Enterprises are intimidated by this requirement even when it makes total sense to have it. That will open the door for them to ask "what are these steps" and we have an offline document we share with the customer in such cases.
@adisht what do you think?

Reply:
Some additional detail here would be very useful.  Specific operations we would mention as examples, perhaps?  (Also, is this something we could add in a future update/ticket?

Reply:
Yes we can do that on a future ticket

Reply:
See Jira DOC-720 to track progress.